scribus/text/storytext.cpp

```diff
@@ -198,21 +199,22 @@ void StoryText::moveCursorLeft(bool isGrapheme)
 {
 	if (isGrapheme)
 	{
-		int pos;
-		QTextBoundaryFinder boundaryFinder(QTextBoundaryFinder::Grapheme, plainText());
+		BreakIterator* it = getGraphemeIterator();
+		if (!it)
+			return;
+
+		int pos = cursorPosition();
+		it->setText(plainText().utf16());
```

Comment:
I tried this but cases a lot of problem. Iterator starts mix text and never work with CJK. I think the problem that we use static vars which is unsafe thread. So better leave it as it unless there is somebody willing to dig ICU source code to find the problem.